app/js/utils/helpers.js

```diff
@@ -118,7 +118,19 @@ export const getConceptShortName = (concept, locale) => {
         );
       }
       if (!foundConcept) {
-        // could not find any locale preferred name
+        // attempt to find the preferred short name in English
```

Comment:
Search now does the following order:
* the preferred SHORT name in the current locale
* any SHORT name in the current locale
* any preferred name in the current locale
* preferred short name in English
* any short name in English
* concept.display

Happy to discuss if we want.